segmentstore/server/src/main/java/io/pravega/segmentstore/server/store/ServiceBuilder.java

```diff
@@ -241,6 +241,14 @@ public void initialize() throws DurableDataLogException {
         getSingleton(this.containerManager, this.segmentContainerManagerCreator).initialize();
     }
 
+    /**
+     * To get the storageFactory after a ServiceBuilder has been initialized.
+     * @return StorageFactory instance used to initialize ServiceBuilder.
+     */
+    public StorageFactory getStorageFactory() {
+        return createStorageFactory();
+    }
+
```

Comment:
You don't need this method , why not just call createStorageFactory directly ?

Reply:
Ok. Done.